Model/ProductStatusAdapter.php

```diff
@@ -91,7 +91,7 @@ public function disableProductWithSku($sku)
         $this->validateSku($sku);
         try {
             $product = $this->productRepository->get($sku);
-            if ($product->getStatus() === ProductStatus::STATUS_DISABLED) {
+            if ($product->getStatus() == ProductStatus::STATUS_DISABLED) {
```

Comment:
Ah yes, the value returned from the Products are strings, but the constant is an int. I caught that one in one of the first versions of the module, too. Thanks!